bundle/manifests/sailoperator.clusterserviceversion.yaml

```diff
@@ -776,7 +776,7 @@ spec:
                 - --zap-log-level=info
                 command:
                 - /sail-operator
-                image: quay.io/sail-dev/sail-operator:1.25-latest
+                image: quay.io/sail-dev/sail-operator:1.25
```

Comment:
Don't we need 1.25.0 tag?

Reply:
Yes... Do we have minor version tags as well? i.e. both `1.25.0` and a floating `1.25` tag?

Reply:
I updated to just be `VERSION` for now. If we want floating tags later for MINOR_VERSION that can be changed in `main`.